docs/docs/reference/crd-reference/metricsprovider.md

```diff
@@ -102,50 +102,52 @@ For detailed information please look at the [Examples section](#examples).
 
 ## Examples
 
-### Prometheus
-
-An example of Prometheus as a metrics provider with a Secret holding
-the authentication data looks like the following:
-
-```yaml
-{% include "./assets/keptnmetricsprovider-prometheus.yaml" %}
-```
-
-> **Note**
-Setting the `.spec.secretKeyRef.key` field in `KeptnMetricsProvider` is not necessary,
-as `user` and `password` key names are required to be present in the linked Secret.
-Setting this field won't have any effect.
-
-### Datadog
-
-An example of Datadog as a metrics provider with a Secret holding
-the authentication data looks like the following:
-
-```yaml
-{% include "./assets/keptnmetricsprovider-datadog.yaml" %}
-```
-
-> **Note**
-Setting the `.spec.secretKeyRef.key` field in `KeptnMetricsProvider` is not necessary,
-as `DD_CLIENT_API_KEY` and `DD_CLIENT_API_KEY` key names must be
-present in the linked Secret.
-Setting this field has no effect.
-
-### Dynatrace and DQL
-
-An example of Dynatrace as a metrics provider with a Secret holding
-the authentication data looks like the following:
-
-```yaml
-{% include "./assets/keptnmetricsprovider-dynatrace.yaml" %}
-```
-
-> **Note**
-When using Dynatrace as metrics provider you can
-define the key name of your DT token stored in a secret,
-which is not possible for Datadog or Prometheus.
-For this example `myCustomTokenKey` was used.
-
+<!-- markdownlint-disable MD046 -->
+<!-- markdownlint-disable MD051 -->
+=== "Prometheus"
```

Comment:
Let's try to use a html link tag here, therefore we might be able to remove the MD051 ignorer [here](https://github.com/keptn/lifecycle-toolkit/pull/3005/files#diff-fc8617bb6853a0241142d24001ab35b560255f3a7adb0962f615f41c5b453d0fR106). More info [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/md051.md)

Reply:
Thank you for the review @odubajDT
Removed the MD051 ignored all together from this page.